### PR TITLE
Dark mode theme colors for declarative charts

### PR DESCRIPTION
**Pull request by srmukher**
Dark mode theme colors for declarative charts.

### CODE REVIEW

File: packages/charts/react-charting/src/components/DeclarativeChart/DeclarativeChart.tsx [hunk 1]
```diff
@@ -25,6 +25,13 @@ import { GaugeChart } from '../GaugeChart/index';
 import { GroupedVerticalBarChart } from '../GroupedVerticalBarChart/index';
```
Comment (fabricteam) [inline packages/charts/react-charting/src/components/DeclarativeChart/DeclarativeChart.tsx:25]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33439&commitId=a764093fcde34329f9229a1de32a81a3ee86cb1a&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Beak 25.chromium.png|2186| Changed|
|Callout.Rendering callout attached to a rectangle.chromium.png|1493| Changed|
|Callout.No callout width specified.chromium.png|2319| Changed|
|Callout.Top center.chromium.png|2178| Changed|
|Callout.Bottom left edge.chromium.png|2309| Changed|

</details>


  <details><summary><b>react-charting-VerticalBarChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-VerticalBarChart.Basic - Secondary Y Axis.chromium.png|4| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by AtishayMsft** [comment]
change file is missing

File: packages/charts/react-charting/src/components/DeclarativeChart/DeclarativeChart.tsx [hunk 2]
```diff
@@ -25,6 +25,13 @@ import { GaugeChart } from '../GaugeChart/index';
...
 import { VerticalBarChart } from '../VerticalBarChart/index';
 
+import { useTheme } from '@fluentui/react';
+
+export const UseIsDarkTheme = (): boolean => {
+  const theme = useTheme();
+  return theme.isInverted;
```
Comment (AtishayMsft) [inline packages/charts/react-charting/src/components/DeclarativeChart/DeclarativeChart.tsx:32]: is theme guaranteed to be present and not undefined

Comment (srmukher) [reply packages/charts/react-charting/src/components/DeclarativeChart/DeclarativeChart.tsx:32]: will add a check

**Pull request merged**